services/integration/src/routes/constitution/processes.js

```diff
@@ -0,0 +1,8 @@
+const Router = require('express').Router();
```

Comment:
This file also should be called as process.js

Reply:
i fixed many things about files